src/dashboard/alerts/alert.py

```diff
@@ -3,15 +3,15 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath("../.."))
+sys.path.insert(0, os.path.abspath("../../.."))
```

Comment:
Was this necessary to get the imports to work? I had to include it when the top level directory wasn't a package, but it shouldn't be necessary now that it is. Not an issue either way tho, more path entries can't hurt.

Reply:
Fixed